packages/mermaid/src/diagrams/git/gitGraphRenderer.js

```diff
@@ -606,8 +665,12 @@ const drawBranches = (svg, branches) => {
       line.attr('x1', pos);
       line.attr('y2', maxPos);
       line.attr('x2', pos);
+    } else if (dir === 'BT') {
+      line.attr('y1', maxPos);
+      line.attr('x1', pos);
+      line.attr('y2', 30);
```

Comment:
Let's avoid using [Magic numbers](https://stackoverflow.com/questions/47882/what-are-magic-numbers-and-why-do-some-consider-them-bad) in new commits.